File: PRAK401-2210817220011-MeldianaHandayani.py
print("kelipatan = ", end=' ')
jumlah= int(input())
print("simbol    = ", end=' ')
klp= (input())
for i in range(1, 51):
  if i %jumlah !=0:
         print('%d'% i, end=' ')
  if i % jumlah ==0:
        print('%c'% klp, end=' ')
  

File: PRAK401-2210817220011-MeldianaHandayani.c
#include <stdio.h>
int main(){
printf("kelipatan = ");
    char KLP;
    int i, jumlah, hasil;
scanf("%i %c", &jumlah, &KLP);
     for(int i =1 ; i<=50; i++){
     if (i %jumlah !=0 ){
         printf("%d ", i);
             }printf("", i);
        if(i % jumlah ==0){
        printf("%c ", KLP);
             } printf("", KLP);  
     } 
}